package.json

```diff
@@ -14,16 +14,19 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^4.1.0",
+    "@angular/cdk": "github:angular/cdk-builds",
```

Comment:
new cdk-builds from material: https://material.angular.io/guide/getting-started

Reply:
That looks cool.